src/icon.rs

```diff
@@ -212,7 +256,7 @@ impl Icons {
         m.insert("gform", "\u{f298}"); // ""
         m.insert("gif", "\u{f1c5}"); // ""
         m.insert("git", "\u{f1d3}"); // ""
-        m.insert("go", "\u{e626}"); // ""
+        m.insert("go", "\u{e627}"); // ""
```

Comment:
I remember we have a issue before discussing this `Go` icon seems a little bit creepy for the eyes in some background, maybe we could keep the origin one

Reply:
ref: https://github.com/Peltoche/lsd/pull/333
PS: I don't have any strong opinions on this anymore